cscs-checks/prgenv/helloworld.py

```diff
@@ -155,9 +164,7 @@ def __init__(self, lang, linkage):
 
         # On SLURM there is no need to set OMP_NUM_THREADS if one defines
         # num_cpus_per_task, but adding for completeness and portability
-        self.variables = {
-            'OMP_NUM_THREADS': str(self.num_cpus_per_task)
-        }
+        self.variables['OMP_NUM_THREADS']: str(self.num_cpus_per_task)
```

Comment:
Keep this as it is and define the workaround as a separate hook after this one. The workaround should do `self.variables.update(...)`.